cpp/tests/test_glct_secir.cpp

```diff
@@ -0,0 +1,525 @@
+
+        model->parameters.get<mio::glsecir::TransmissionProbabilityOnContact>() = 0.05;
+
+        mio::ContactMatrixGroup& contact_matrix = model->parameters.get<mio::glsecir::ContactPatterns>();
+        contact_matrix[0]                       = mio::ContactMatrix(Eigen::MatrixXd::Constant(1, 1, 10));
+        contact_matrix[0].add_damping(0.7, mio::SimulationTime(2.));
+
+        model->parameters.get<mio::glsecir::RelativeTransmissionNoSymptoms>() = 0.7;
+        model->parameters.get<mio::glsecir::RiskOfInfectionFromSymptomatic>() = 0.25;
+        model->parameters.get<mio::glsecir::Seasonality>()                    = 0.;
+        model->parameters.get<mio::glsecir::StartDay>()                       = 0;
+
+        // --- Define initial distribution of the population in the subcompartments. ---
+        std::vector<std::vector<ScalarType>> initial_populations = {
+            {750},
+            {30, 20},
+            {20 * StartingProbabilitiesInfectedNoSymptoms[0], 10 * StartingProbabilitiesInfectedNoSymptoms[0],
+             10 * StartingProbabilitiesInfectedNoSymptoms[0], 20 * (1 - StartingProbabilitiesInfectedNoSymptoms[0]),
+             10 * (1 - StartingProbabilitiesInfectedNoSymptoms[0]),
+             10 * (1 - StartingProbabilitiesInfectedNoSymptoms[0])},
+            {50 * StartingProbabilitiesInfectedSymptoms[0], 50 * (1 - StartingProbabilitiesInfectedSymptoms[0])},
+            {50 * StartingProbabilitiesInfectedSevere[0], 50 * (1 - StartingProbabilitiesInfectedSevere[0])},
+            {10 * StartingProbabilitiesInfectedCritical[0], 10 * StartingProbabilitiesInfectedCritical[0],
+             5 * StartingProbabilitiesInfectedCritical[0], 3 * StartingProbabilitiesInfectedCritical[0],
+             2 * StartingProbabilitiesInfectedCritical[0], 10 * (1 - StartingProbabilitiesInfectedCritical[0]),
+             10 * (1 - StartingProbabilitiesInfectedCritical[0]), 5 * (1 - StartingProbabilitiesInfectedCritical[0]),
+             3 * (1 - StartingProbabilitiesInfectedCritical[0]), 2 * (1 - StartingProbabilitiesInfectedCritical[0])},
+            {20},
+            {10}};
+        // Transfer the initial values in initial_populations to the model->
+        std::vector<ScalarType> flat_initial_populations;
+        for (auto&& vec : initial_populations) {
+            flat_initial_populations.insert(flat_initial_populations.end(), vec.begin(), vec.end());
+        }
+        for (size_t i = 0; i < LctState::Count; i++) {
+            model->populations[mio::Index<LctState>(i)] = flat_initial_populations[i];
+        }
+    }
+
+    virtual void TearDown()
+    {
+        delete model;
+    }
+
+public:
+    Model* model = nullptr;
+};
+
+// Test compares a simulation with a previous output of an LCT model stored in a .csv file.
+// This tests that the GLCT model reduces to an LCT model with the default parameters as well as nothing has changed
```

Comment:
If you are using the default parameters, then why do you set them manually? Or did you set them according to the values used in the LCT model in the corresponding test?

Reply:
Added comment.